docs/pages/getting_started.rst

```diff
@@ -1,55 +1,309 @@
 
-There are two primary methods for iterating through submissions.
+Get submissions from a subreddit
+--------------------------------
 
-The first method is to pick a subreddit, and then iterate through one of its
-many generators of submissions, e.g., ``hot``, ``new``, or ``top``. In the
-following example we build a list of submissions which may be used for later
-processing.
+Now that you have a ``Subreddit`` instance, you can get some submissions
+(``Submission`` instances) from it! There are several ways of sorting all the
+submissions of a subreddit: hot, new, top, etc. A ``Subreddit`` instance has a
+method for each of these sorting approaches, namely, these:
+
+    ``controversial``
+    ``gilded``
+    ``hot``
+    ``new``
+    ``rising``
+    ``top``
+
+Each of these methods will return ``ListingGenerator``, something that you can
+iterate through. For example:
 
 .. code-block:: python
 
-   submissions = []
-   for submission in reddit.subreddit('python').hot():
-       submissions.append(submission)
+    # assuming you have a Subreddit instance referenced by subreddit
+    for submission in subreddit.hot(limit=10):
+        print(submission.title) # Output: the title of the submission
+        print(submission.ups) # Output: upvote count
+        print(submission.downs) # Output: downvote count
+        print(submission.id) # Output: the ID of the submission
+        print(submission.url) # Output: the URL of the submission
 
-.. note:: Just like on the website, multiple subreddits can be specified by
-   joining them with a plus (`+`), e.g., ``python+redditdev``.
 
-The second method is to iterate over submissions from a subreddit's submission
-stream (:meth:`praw.models.reddit.subreddit.SubredditStream.submissions`). A
-submission stream indefinitely yields new submissions as they are made to
-Reddit. For example:
+You can create ``Submission`` instances in other ways too:
 
 .. code-block:: python
 
-   subreddit = reddit.subreddit('python')
-   for submission in subreddit.stream.submissions():
-       print(submission.selftext)
+    # assuming you have a Reddit instance referenced by reddit
+    submission = reddit.submission(id="39zje0")
+    print(submission.title) # Output: reddit will soon only be available ...
+
+    # or
+    submission = reddit.submission(url="https://www.reddit.com/......")
+
+
+Get redditors
+-------------
+
+There are several ways to get a redditor (a ``Redditor`` instance), two of the
+most common ones are:
 
-The ``selftext`` attribute of a :class:`.Submission` contains the markdown
-formatted text of a *self* type submission. For a complete list of attributes,
-and their values, available on a PRAW object try the following:
+    - via the ``author`` attribute of a ``Submission`` instance
+    - call the ``redditor`` method on a ``Reddit`` instance
+
+For example:
 
 .. code-block:: python
 
-   from pprint import pprint
-   pprint(vars(submission))
```

Comment:
Yes. This is not the final version of getting started. I can add a section dedicated on attributes and methods available. How about that?